Mathlib/Condensed/Light/Discrete.lean

```diff
@@ -0,0 +1,57 @@
+/-
+Copyright (c) 2023 Dagur Asgeirsson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Dagur Asgeirsson
+-/
+import Mathlib.CategoryTheory.Sites.ConstantSheaf
+import Mathlib.CategoryTheory.Sites.Equivalence
+import Mathlib.CategoryTheory.Sites.LeftExact
+import Mathlib.Condensed.Light.Basic
+
+/-!
+
+# Discrete-underlying adjunction
+
+Given a well-behaved concrete category `C`, we define a functor `C ⥤ Condensed C` which associates
+to an object of `C` the corresponding "discrete" condensed object (see `Condensed.discrete`).
+
+In `Condensed.discrete_underlying_adj` we prove that this functor is left adjoint to the forgetful
+functor from `Condensed C` to `C`.
+-/
+
+universe u v w
+
+open CategoryTheory Limits Opposite GrothendieckTopology
+
+variable (C : Type w) [Category.{u} C] [HasSheafify (coherentTopology LightProfinite) C]
```

Comment:
Untested, but I think that a universe annotation would be helpful here, so that Lean does not have to make guesses below:
```suggestion
variable (C : Type w) [Category.{u} C] [HasSheafify (coherentTopology LightProfinite.{u}) C]
```